inference-engine/tests_deprecated/functional/ie_tests/src/custom_matcher.cpp

```diff
@@ -368,7 +368,7 @@ void Regression::Matchers::CustomMatcher::checkResult() {
                 maxAverageDiff = std::max(maxAverageDiff, avgDiff / avgFrames);
 
                 //TODO: add test_log parse from command line
-// #define TEST_LOG
+#define TEST_LOG
```

Comment:
You forgot to add `//`

Reply:
lets wait for error from cpplint then - if not wouldnt fix it now